rum/rum-android-mobile.mdx

```diff
@@ -0,0 +1,294 @@
+---
+title: "Mobile RUM - Android SDK"
+sidebarTitle: "Mobile RUM"
+---
+
+# Overview
+The Mobile RUM SDK provides a customizable suite of tools to analyze and optimize the performance of Android applications. This guide ensures the SDK is initialized and configured for real-time monitoring and telemetry transmission to the Middleware platform throughout the app's lifecycle. Isolate ANR and network changes, quickly detect application crashes, identify slow or frozen frames and more.
+
+<Note> To learn more about the Mobile RUM Android SDK, navigate to our GitHub repository [here](https://github.com/middleware-labs/middleware-android/#middleware-android-sdk). </Note> 
+
+# Prerequisites
+
+1. Android SDK Version `>21`
```

Comment:
Change to hypen based list instead of numbered for consistency across other docs